client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -173,6 +175,17 @@ public void testExists() {
         getAndExistsTest(Request::exists, "HEAD");
     }
 
+    public void testIndicesExist() {
+        String[] indices = IntStream.range(1, randomIntBetween(1, 10))
+            .mapToObj(i -> randomAlphaOfLength(10))
+            .toArray(String[]::new);
```

Comment:
you can use randomIndicesNames

Reply:
Sorry for the potentially dumb question, but where can I find the method? I found one in IndicesRequestIT and one in ESBlobStoreRepositoryIntegTestCase, but I don't think we will be able to use those.

Reply:
I think you need to merge master in, that's all